azkaban-webserver/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -338,6 +340,24 @@ private void handleExecutionFlowPage(HttpServletRequest req,
       page.render();
       return;
     }
+    
```

Comment:
Planning to add a method named 'addExternalAnalyzer'

I will separate out this method to a static utils class(ExternalAnalyzerUtils.java) and add a unit test for it. I hope this sounds good?

Having this method private in the same class will make it tricky to test. I will have to choose one of the four methods listed here. http://www.artima.com/suiterunner/privateP.html

Hence I suggest the above approach. Let me know what you think?